web-src/src/components/SavePromptButton.js

```diff
@@ -149,7 +151,7 @@ export function SavePromptButton(props) {
       ...customPromptTemplates.filter((template) => template.id !== selectedTemplate.id),
       updatedTemplate,
     ];
-    saveTemplates(newCustomPromptTemplates).then(() => {
+    saveTemplates(newCustomPromptTemplates, runMode).then(() => {
```

Comment:
Missing `formatMessage` argument when calling `saveTemplates`. This leads to an uncaught exception as the function is expecting the argument.

Reply:
Good catch, @askayastha22 !